codelab-master/apps/playground/src/app/quiz/components/question/question.component.scss

```diff
@@ -0,0 +1,169 @@
+@import "variables";
+
+.options {
+  margin: 0 0 10px 0.4rem;
+  padding: 5px;
+  width: 39.5rem !important;
+
+  mat-radio-button, mat-checkbox {
+    cursor: pointer;
+    width: 39.5rem !important;
```

Comment:
this is a really weird value. 
If we're using material styles, maybe let's not override the styles to keep it consistent with other parts of the app?

Reply:
I changed the value here to 630px, in QuizComponent scss and also in styles.scss.